tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "lib": ["ESNext", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
+    "noEmit": true,
```

Comment:
Setting this because I wanted to `npx tsc -p .` to check that everything typechecked and did not want to emit the JS files

Reply:
Just make sure the build command still works.

Reply:
I ran it which produced different `react.js.map` files which are fairly inscrutable. Lmk if there's something else I should check.